### PR TITLE
Updates to fedora ansible guide.

### PR DESCRIPTION
**Pull request by aveshagarwal**
Updates to fedora ansible guide.

### CODE REVIEW

**Comment by aveshagarwal** [comment]
@eparis

File: docs/getting-started-guides/fedora/fedora_ansible_config.md [hunk 1]
```diff
@@ -29,151 +29,130 @@ Configuring kubernetes on Fedora via Ansible offers a simple way to quickly crea
...
+    node1 = kube-node-01.example.com
+    node2 = kube-node-02.example.com
 ```
 
-**Make sure your local machine** 
+**Make sure your local machine has**
 
- - has ansible
- - has git
+ - ansible (preferably 1.9+)
```
Comment (eparis) [inline docs/getting-started-guides/fedora/fedora_ansible_config.md]: 1.9.0 is a requirement, not a preference

**Comment by eparis** [comment]
fix the one comment, then LGTM

File: docs/getting-started-guides/fedora/fedora_ansible_config.md (hunk 1 above)
Comment (aveshagarwal) [inline docs/getting-started-guides/fedora/fedora_ansible_config.md]: ok fixed, and updated the PR.

**Comment by bgrant0607** [comment]
@eparis Is this ready? 

We're implementing a number of sweeping doc transformation passes, so this will need to be rebased and have hack/run-gendocs.sh applied again at some point before we can merge it.

**Comment by eparis** [comment]
LGTM.  Avesh, please update

**Comment by bgrant0607** [comment]
I think we're done with our new doc transformation passes. Please rebase and run hack/run-gendocs.sh. If this is done over the weekend, we can get the updates into 1.0 docs on kubernetes.io.

**Comment by aveshagarwal** [comment]
@bgrant0607 I will get it done today.

**Comment by aveshagarwal** [comment]
I have rebased and resolved all conflicts. However, running hack/run-gendocs.sh outputs following error:

kubectl-dash-f:
231: target file "apache.json" does not exist

Can I ignore it? The file apache.json is being asked to create in the guide so does not pre-exist.

**Comment by bgrant0607** [comment]
Please prefix apache.json with `/tmp/` so that the checker knows that the file isn't one from the source tree.

**Comment by aveshagarwal** [comment]
@bgrant0607 thanks. I have pushed updated PR and hopefully it should be good to go now.

**Comment by bgrant0607** [comment]
Indeed, all tests passed. We could autogenerate the TOC, but that can be done later. LGTM.

**Pull request merged**